_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -0,0 +1,2544 @@
+
+class DataImportDependencyProvider extends SprykerDataImportDependencyProvider
+{
+    protected function getDataImporterPlugins(): array
+    {
+        return [
+            new StoreDataImportPlugin(),
+            new CountryStoreDataImportPlugin(),
+            new CurrencyStoreDataImportPlugin(),
+            new LocaleStoreDataImportPlugin(),
+            new DefaultLocaleStoreDataImportPlugin(),
+        ];     
+    }
+}
+```
+
+### Adjust ConsoleDependencyProvider.
+
+```
+src/Pyz/Zed/Console/ConsoleDependencyProvider.php
+```
+
+```php
+<?php
+namespace Pyz\Zed\Console;
+
+use Spryker\Zed\Console\ConsoleDependencyProvider as SprykerConsoleDependencyProvider;
+use Spryker\Zed\DataImport\Communication\Console\DataImportConsole;
+use Spryker\Zed\StoreDataImport\StoreDataImportConfig;
+use Spryker\Zed\Locale\Communication\Plugin\Application\LocaleApplicationPlugin;
+use Spryker\Zed\LocaleDataImport\LocaleDataImportConfig;
+use Spryker\Zed\CountryDataImport\CountryDataImportConfig;
+use Spryker\Zed\CurrencyDataImport\CurrencyDataImportConfig;
+
+/**
+ * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+ * @method \Pyz\Zed\Console\ConsoleConfig getConfig()
+ */
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+    /**
+     * @var string
+     */
+    protected const COMMAND_SEPARATOR = ':';
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Symfony\Component\Console\Command\Command>
+     */
+    protected function getConsoleCommands(Container $container): array
+    {
+        return [
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . CountryDataImportConfig::IMPORT_TYPE_COUNTRY_STORE),
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . CurrencyDataImportConfig::IMPORT_TYPE_CURRENCY_STORE),
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . LocaleDataImportConfig::IMPORT_TYPE_LOCALE_STORE),
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . LocaleDataImportConfig::IMPORT_TYPE_DEFAULT_LOCALE_STORE),
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . StoreDataImportConfig::IMPORT_TYPE_STORE),            
+        ];
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Shared\ApplicationExtension\Dependency\Plugin\ApplicationPluginInterface>
+     */
+    public function getApplicationPlugins(Container $container): array
+    {
+        $applicationPlugins = parent::getApplicationPlugins($container);
+
+        $applicationPlugins[] = new LocaleApplicationPlugin();
+
+        return $applicationPlugins;
+    }
+
+}
+```
+
+
+### Preparing csv files for configure stores, locales, currencies, countries via data import 
```

Comment:
those are examples. Mb we can mention that they can get all this data from our demoshops/suite?